.github/workflows/release.yml

```diff
@@ -1,26 +1,46 @@
 ---
 name: Release
 on:
-  push:
-    tags:
-      - 'v*'
+  workflow_dispatch:
+    inputs:
+      release:
+        description: 'Release tag'
+        required: true
+      previous-tag:
+        description: 'Previous release tag'
+        required: true
 jobs:
   release:
     name: Release
+    if: ${{ github.repository == 'shipwright-io/build' }}
     runs-on: ubuntu-latest
+    permissions:
+      contents: write # To be able to update releases
     steps:
     - name: Checkout
       uses: actions/checkout@v2
       with:
+        ref: ${{ github.event.inputs.release }}
         fetch-depth: 0
     - name: Set up Go
       uses: actions/setup-go@v2
       with:
         go-version: 1.17.x
+    - name: Build Release Changelog
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        PREVIOUS_TAG: ${{ github.event.inputs.previous-tag }}
+      # This creates a set of release notes at Changes.md
+      run: |
+        # might not be necessary but make sure
+        chmod +x "${GITHUB_WORKSPACE}/.github/draft_release_notes.sh"
+        export GITHUB_TOKEN
+        export PREVIOUS_TAG
+        "${GITHUB_WORKSPACE}/.github/draft_release_notes.sh"
     - name: Run GoReleaser
       uses: goreleaser/goreleaser-action@v2
       with:
         version: latest
-        args: release --rm-dist
+        args: release --rm-dist --release-notes Changes.md
```

Comment:
Pass the argument that lets us override goreleaser's default set of release notes.